src/me/ryanhamshire/PopulationDensity/PlayerEventHandler.java

```diff
@@ -37,14 +37,16 @@
 public class PlayerEventHandler implements Listener {
 	private DataStore dataStore;
 	PopulationDensity instance;
+	private boolean teleportNearbyAnimals;
 
 	// queue of players waiting to join the server
 	public ArrayList<LoginQueueEntry> loginQueue = new ArrayList<LoginQueueEntry>();
 
 	// typical constructor, yawn
-	public PlayerEventHandler(DataStore dataStore, PopulationDensity plugin) {
+	public PlayerEventHandler(DataStore dataStore, PopulationDensity plugin, boolean teleportNearbyAnimals) {
```

Comment:
Um... this is not a good idea for any config option.

Reply:
> this is not a good idea

Or rather, this does not follow the conventions of the rest of this plugin afaik